javascript/javascript1/week3/ex-5.js

```diff
@@ -0,0 +1,28 @@
+// Exercise 5: CactusIO-interactive (Smart phone usage app) optional
+
+const activities = [];
+const usageLimit = 75;
+
+function addActivity(date, activity, duration) {
+	const actualDate = new Date().toLocaleDateString("dk-DK");
+	activities.push({ date: actualDate, activity, duration });
+}
+
+function showStatus(arr = activities) {
+	let totalDuration = 0;
+	arr.map(({ duration }) => (totalDuration += duration));
+	console.log(
+		`You have added ${activities.length} activities. They amount to ${totalDuration} min. of usage`
+	);
+
+	totalDuration > usageLimit
```

Comment:
Good job on making this as concise as it is! ⭐ The assignment also asks for a new feature to be implemented, but I will not ask that of you as the task is optional; unless you want to, of course! 😛 

A minor styling nitpick is that I would prefer an if statement here since the branches of your ternary are statements and not expressions; this is a personal opinion though. Alternatively, the ternary could also be moved into the `console.log` so it returns a string instead. 😄